lab1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="app.css">
+	<title>Youtube Famous</title>
+</head>
+<body>
+		<header>
+			<h1>View Collector</h1>
+		</header>
+		
+		<div class="story">
+			<h4>The Story Goes Here</h4>
+		</div>
+
+		<div class="container">
+			<div class="nav">
+			<h3>Views: 1337 </h3>
+			</div>
+
+			<button class="starter">Get Views</button>
+		</div>
+
+
+		<div class="container">
+		
+			<div class="generator">
+				<h3 class="center">Educational Video</h3>
+
+				<p class="center">Teach your fellow viewers about anyting for a rewarding 10 views per minute!</p>
+
+				<button class="buttoninside">10 Views</button>
```

Comment:
Missing rate (how fast a generator generates value) (the number on the bottom left)